iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -52,33 +52,24 @@ -(BOOL)activityTypeValid:(NSString*)activityType
 
 -(id)isSupported:(id)unused
 {
-    if ([TiUtils isIOS8OrGreater] == YES) {
-        return NUMBOOL(_supported);
-    } else {
-        return NUMBOOL(NO);
-    }
+    return NUMBOOL(_supported);
 }
 
 -(BOOL) determineMinRequirements:(NSDictionary *)props
 {
     _isValid = NO;
-    if([TiUtils isIOS8OrGreater]){
-        if([props objectForKey:@"activityType"]){
-            if(![self activityTypeValid:[TiUtils stringValue:@"activityType" properties:props]]){
-                DebugLog(@"[ERROR] activityType provided is not defined in your projects tiapp.xml file");
-                return NO;
-            }else{
-                _isValid = YES;
-                return YES;
-            }
-        }else{
-            DebugLog(@"[ERROR] activityType property is required on creation");
+    if ([props objectForKey:@"activityType"]) {
+        if (![self activityTypeValid:[TiUtils stringValue:@"activityType" properties:props]]) {
+            DebugLog(@"[ERROR] activityType provided is not defined in your projects tiapp.xml file");
             return NO;
+        } else {
+            _isValid = YES;
+            return YES;
         }
+    } else {
+        DebugLog(@"[ERROR] activityType property is required on creation");
+        return NO;
     }
```

Comment:
The `else` is not needed, since both inner-statements of the `if` statement return already. Just write it as a default value:
```objc
if (...) {

}

DebugLog(@"[ERROR] activityType property is required on creation");
return NO;
```